lmdeploy/pytorch/models/q_modules.py

```diff
@@ -98,18 +101,23 @@ def __init__(self,
             self.register_parameter('bias', None)
 
     @classmethod
-    def from_float(cls, mod):
+    def from_float(cls, 
+                   mod: nn.Module,
+                   initialization: bool = True):
         """Class method to create a QLinear instance from a floating-point
-        module."""
+        module. initialization for dummy init."""
```

Comment:
It seems that `initialization = True` means real init and `initialization = False` means dummy init?